### PR TITLE
Add lastUpdated field to endpoint status

### PR DESCRIPTION
**Pull request by mpolden**

### CODE REVIEW

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/ApplicationSerializer.java [hunk 1]
```diff
@@ -445,9 +447,12 @@ private DeploymentMetrics deploymentMetricsFromSlime(Inspector object) {
...
     private RotationStatus rotationStatusFromSlime(Inspector parentObject) {
         var object = parentObject.field(rotationStatusField);
-        var statusMap = new LinkedHashMap<RotationId, Map<ZoneId, RotationState>>();
+        var statusMap = new LinkedHashMap<RotationId, RotationStatus.Targets>();
         object.traverse((ArrayTraverser) (idx, statusObject) -> statusMap.put(new RotationId(statusObject.field(rotationIdField).asString()),
-                                                                              singleRotationStatusFromSlime(statusObject.field(statusField))));
+                                                                              new RotationStatus.Targets(
+                                                                                      singleRotationStatusFromSlime(statusObject.field(statusField)),
+                                                                                      // TODO(mpolden): Make non-optional after October 2019
+                                                                                      Serializers.optionalInstant(statusObject.field(lastUpdatedField)).orElse(Instant.EPOCH))));
```
Comment (jonmv) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/ApplicationSerializer.java]: Could just make an instance from the field, without optionality — non-existent value is 0, which is EPOCH.

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java [hunk 2]
```diff
@@ -702,11 +702,6 @@ private void toSlime(Cursor object, Instance instance, DeploymentSpec deployment
                 .map(URI::toString)
                 .forEach(globalRotationsArray::addString);
 
-        instance.rotations().stream()
-                .map(AssignedRotation::rotationId)
-                .findFirst()
-                .ifPresent(rotation -> object.setString("rotationId", rotation.asString()));
```
Comment (jonmv) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java]: I think this is used.

**Review by jonmv — APPROVED** [review]
Approved, given you remove the usage of `rotationId` :)

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java (hunk 2 above)
Comment (mpolden) [reply controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java]: Removed that commit.

**Comment by jonmv** [comment]
Hehehe, ok :)

**Pull request merged**